src/modrover/main.py

```diff
@@ -1,32 +1,40 @@
+# TODO: Deprecated, delete when done
+
 from operator import attrgetter
 from pathlib import Path
 from typing import Dict, List, Optional
 
 import pandas as pd
 from pplkit.data.interface import DataInterface
 
-from .info import RoverSpecs
-from .modelhub import ModelHub
+from .learner import Learner
 from .strategies import strategy_type_dict
 
 
 class Rover:
 
-    def __init__(self, specs: RoverSpecs, modelhub: ModelHub):
-        self.specs = specs
-        self.modelhub = modelhub
-        self.strategies = [
-            strategy_type_dict[name](self.modelhub)
-            for name in self.specs.strategy_names
-        ]
+    def __init__(self, num_covariates: int):
+        self.performances = {}
```

Comment:
Should we called it `learners` rather than `performances`? Since it is a dictionary store all the learners.